### ISSUE TITLE
There's no way to play music over more than one room without constant interruptions, unless stage music is removed altogether

### ISSUE THREAD

**Issue by Meowlala** [Meowlala/RepentanceAPIIssueTracker#57]
Playing and / or queueing a track through MusicManager works within a single room, but when you enter another, that room / the current floor's track takes over, requiring you to restart your track from the beginning. Removing and reimplementing the stage music can resolve this, which is what StageAPI does to make its custom floors' music function.

Ideally, we could override the music in a way that doesn't result in the game trying to take over immediately. It might also be nice to be able to start music from an arbitrary point in the track, rather than exclusively the beginning, if possible.

**Comment by jsgnextortex** [comment]
Maybe a separate function or an extra parameter in the MusicMan:Play function to determine if it's room music or a stage music, since there are cases where you may want to play something just for a room.

**Comment by NAT0P0TAT0** [comment]
YES, would love an extra parameter in Play/Queue/Crossfade/Fadein that sets the music to play kinda like a layer, coming in at a specific timestamp (based on how long its been since the music last started normally) rather than starting at the beginning.
